src/model/model.ts

```diff
@@ -266,11 +257,7 @@ export class Model {
   }
 
   // getValuesForFieldInPolicy gets all values for a field for all rules in a policy, duplicated values are removed.
-  public getValuesForFieldInPolicy(
-    sec: string,
-    key: string,
-    fieldIndex: number
-  ): string[] {
+  public getValuesForFieldInPolicy(sec: string, key: string, fieldIndex: number): string[] {
```

Comment:
I already fixed this in: https://github.com/casbin/node-casbin/commit/7e78cbaa9b4461cb7f241743fa83c4afcd107be8. Please remove it.